### ISSUE TITLE
Database migrations

### ISSUE THREAD

**Issue by izi** [fhirbase/fhirbase-plv8#80]
The releases 7/8 of beta are fixed to public schema.

The first statement in the file is:

```
CREATE SCHEMA IF NOT EXISTS public; SET search_path TO public;
```

This effectively kills any possibility to run install script in context of different schemas.

**Comment by danil** [comment]
@izi

> The releases 7/8 of beta are fixed to public schema.
> 
> The first statement in the file is:
> 
> CREATE SCHEMA IF NOT EXISTS public; SET search_path TO public;
> 
> This effectively kills any possibility to run install script in
> context of different schemas.

fixed, please try v0.0.1-beta.10

**Comment by izi** [comment]
It works fine. Thanks.
